File: exhibition-timeslot-booking/src/main/java/org/example/exhibitiontimeslotbooking/common/utils/pageable/SortFields.java
package org.example.exhibitiontimeslotbooking.common.utils.pageable;

import java.util.Set;

public final class SortFields {
    private SortFields() {}
    public static final Set<String> VENUE_SORT = Set.of("id", "name", "createdAt", "updatedAt");
    public static final Set<String> EXHIBITION_SORTS = Set.of("id", "title", "createdAt", "updatedAt");
}

File: exhibition-front/src/types/common/utils/pageable/SortFields.ts
export type VenueSortType = "id" | "name" | "createdAt" | "updatedAt";
export type ExhibitionSortType = "id" | "title" | "createdAt" | "updatedAt";

export type SortDirection = "desc" | "asc";

export type VenueSort = `${VenueSortType},${SortDirection}`;

export type ExhibitionSort = `${ExhibitionSortType},${SortDirection}`;
